portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase

```diff
@@ -177,15 +184,16 @@ definition {
 	@priority = "4"
 	@refactorneeded
 	test AddWebContentArticleWithImageViaURL {
-		Navigator.openURL();
+		JSONWebcontent.addWebContent(
+			content = "WC WebContent Content",
+			groupName = "Test Site Name",
+			title = "Web Content Title");
 
-		ProductMenu.gotoPortlet(
-			category = "Content & Data",
-			portlet = "Web Content");
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "test-site-name");
 
-		WebContentNavigator.gotoAddCP();
+		WebContent.viewTitle(webContentTitle = "Web Content Title");
 
-		PortletEntry.inputTitle(title = "Web Content Title");
+		WebContentNavigator.gotoEditCP(webContentTitle = "Web Content Title");
 
 		ItemSelector.gotoItemSelectorViaCKEditor();
 
```

Comment:
CanAddHyperlinkToExistingText test does not need to JSON a new site since it's now initialized in global setup
 
Remove `JSONGroup.addGroup(groupName = "Test Site Name");`

Also replace the `WebContent.viewTitle(webContentTitle = "Test Web Content");` with a verifyElementPresent